src/Spectre.Console/Widgets/TextPath.cs

```diff
@@ -52,12 +52,12 @@ public TextPath(string path)
         _parts = path.Split(new char[] { '/' }, StringSplitOptions.RemoveEmptyEntries);
 
         // Rooted Unix path?
-        if (path.StartsWith("/"))
+        if (path.StartsWith("/", StringComparison.Ordinal))
```

Comment:
```suggestion
        if (path.StartsWith('/'))
```
use the char-overload?

Reply:
This overload is not available in netstandard2.0. I can add an extension method in `StringExtensions.cs` if you want

````
#if NETSTANDARD2_0
internal static bool StartsWith(this str, char c) => str.Length > 0 && str[0] == 'c';
#endfif
````